### ISSUE TITLE
Compilation & Proving & "How to"

### ISSUE THREAD

**Issue by AtHeartEngineer** [Rate-Limiting-Nullifier/noir-rln#1]
@curryrasul will give more details here

**Comment by curryrasul** [comment]
There was a problem with a compilation of RLN circuits in Noir. However in noir-lang/noir#1848 it was told that it works well with current nightly versions. 
We should do:
* [ ] Compilation test to check if everything's working well
* [ ] Proving & setting .sh scripts to run proving easy (with generation of inputs)
* [ ] Instruction in readme on "How to run" (as [here](https://github.com/Rate-Limiting-Nullifier/rln-contract) for example)

If anything is wrong with the implementation - try to fix it and/or open other issues with the problem statement and fixing the implementation itself.

**Comment by MuhtasimTanmoy** [comment]
Is this still active / relevant?

**Comment by curryrasul** [comment]
Hey, sure @MuhtasimTanmoy ! The project is not active rn, but if you can update everything, check that everything works, etc. - that'd be very helpful and awesome!

**Comment by AtHeartEngineer** [comment]
> Is this still active / relevant?

@MuhtasimTanmoy @curryrasul @cedoor 
RLN is possibly getting rolled into [Semaphore Protocol](https://github.com/semaphore-protocol)

**Comment by curryrasul** [comment]
@AtHeartEngineer 
Yep, but if we update the repo so that everything works well, that's useful anyway?

**Comment by AtHeartEngineer** [comment]
> @AtHeartEngineer 
> Yep, but if we update the repo so that everything works well, that's useful anyway?

Ya sorry I didn't realize this was the nior repo 🙂
